sdk/src/wallet/account/operations/syncing/transactions.rs

```diff
@@ -101,9 +101,10 @@ where
             if let Some(block_id) = transaction.block_id {
                 match self.client().get_block_metadata(&block_id).await {
                     Ok(metadata) => {
-                        if let Some(inclusion_state) = metadata.ledger_inclusion_state {
-                            match inclusion_state {
-                                LedgerInclusionState::Included => {
+                        // TODO: use tx_state or block_state?
```

Comment:
What's the concern here?

Reply:
Was just confused because BlockState::Pending also mentions `or contains not yet included transaction`
But the same is covered by the TransactionState::Pending, so should be all good how it is, removed the comment